Aptoide/aptoide-uploader
# [Feature/APP-85] Prepare automatic upload

**What does this PR do?**

   Implementation of the Room database for the installed, replaced app removed packages.
   To initiate the service, we will fetch every package installed and insert in the DB, and any changes after that are going to be updates on the DB.

**Where should the reviewer start?**

- InstalledIntentService.kt
- InstallManager.kt
- InstalledRepository.kt
- RoomInstalledPersistence.kt
- InstalledApp.kt

**How should this be manually tested?**

  Install, Remove or Update apps from Aptoide Store and verify the changes on the Room DB using Database Inspector

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APP-85

**Code Review Checklist**
